webviz_subsurface/plugins/_group_tree/_views/_group_tree_view/_view.py

```diff
@@ -0,0 +1,350 @@
+from enum import Enum
+from typing import Any, Dict, List, Optional, Tuple
+
+import webviz_core_components as wcc
+import webviz_subsurface_components as wsc
+from dash import Input, Output, State, callback, html
+from webviz_config.webviz_plugin_subclasses import SettingsGroupABC, ViewABC
+
+from ..._types import NodeType, StatOptions, TreeModeOptions
```

Comment:
Is `_types` only used in the context of this view? If so, consider moving it into the view's folder. #reusability

Reply:
`_types` is used also in the `_utils` folder on plugin level (in the class `EnsembleGroupTreeData`). But maybe this whole `_utils` folder should be moved into the view? The class `EnsembleGroupTreeData` is used in the plugin init function to load the data, but it could be imported from the view. What do you think?